addons/web_editor/static/src/scss/web_editor.variables.scss

```diff
@@ -538,6 +550,26 @@ $o-bg-shapes: ('web_editor': (
     'Bold/11_001': ('position': bottom, 'size': 100% auto, 'colors': (1, 2)),
     'Bold/12': ('position': center, 'size': 100% auto, 'colors': (1, 2, 5)),
     'Bold/12_001': ('position': center, 'size': 100% auto, 'colors': (1, 2, 5)),
+    'Connections/01': ('position': bottom, 'size': 100% var(--ConnectionShape--size), 'colors': (5)),
+    'Connections/02': ('position': bottom, 'size': 100% var(--ConnectionShape--size), 'colors': (5)),
+    'Connections/03': ('position': bottom, 'size': 100% var(--ConnectionShape--size), 'colors': (5)),
+    'Connections/04': ('position': bottom, 'size': 100% var(--ConnectionShape--size), 'colors': (5)),
+    'Connections/05': ('position': bottom, 'size': 100% var(--ConnectionShape--size), 'colors': (5)),
+    'Connections/06': ('position': bottom, 'size': 100% var(--ConnectionShape--size), 'colors': (5)),
+    'Connections/07': ('position': bottom, 'size': 100% var(--ConnectionShape--size), 'colors': (5)),
+    'Connections/08': ('position': bottom, 'size': 100% var(--ConnectionShape--size), 'colors': (5)),
+    'Connections/09': ('position': bottom, 'size': 100% var(--ConnectionShape--size), 'colors': (5)),
+    'Connections/10': ('position': bottom, 'size': 100% var(--ConnectionShape--size), 'colors': (5)),
+    'Connections/11': ('position': bottom, 'size': 100% var(--ConnectionShape--size), 'colors': (5)),
+    'Connections/12': ('position': bottom, 'size': 100% var(--ConnectionShape--size), 'colors': (5)),
+    'Connections/13': ('position': bottom, 'size': 100%, 'colors': (5)),
+    'Connections/14': ('position': bottom, 'size': 100% var(--ConnectionShape--size), 'colors': (5)),
+    'Connections/15': ('position': bottom, 'size': 100% var(--ConnectionShape--size), 'colors': (5)),
+    'Connections/16': ('position': bottom, 'size': 100% var(--ConnectionShape--size), 'colors': (5)),
+    'Connections/17': ('position': bottom, 'size': var(--ConnectionShapeSm--size) 100%, 'colors': (5), 'repeat-x': true),
+    'Connections/18': ('position': bottom, 'size': var(--ConnectionShapeSm--size) 100%, 'colors': (5), 'repeat-x': true),
+    'Connections/19': ('position': bottom, 'size': 100% var(--ConnectionShape--size), 'colors': (5)),
+    'Connections/20': ('position': bottom, 'size': 100% var(--ConnectionShapeLg--size), 'colors': (5)),
```

Comment:
Technically, it’s correct, but I think we can refine it a bit. 😉
I propose:

```scss
$o_shape_connection_height_short: X !default;
$o_shape_connection_regular: Y !default;
$o_shape_connection_tall: Z !default;

.o_we_shape {
    --ShapeConnection--height-short: $o_shape_height_short;
    --ShapeConnection--height-regular: $o_shape_height_regular;
    --ShapeConnection--height-tall: $o_shape_height_tall;
}

$o-bg-shapes: ('web_editor': (
    [...]
    'Connections/20': ('position': bottom, 'size': 100% var(--ShapeConnection--height-tall)),
    [...]
));
```

Breakdown of the proposed changes:

- For default values, use SCSS variables rather than CSS ones. This way these values would be available during the pre-compilation process. It's not strictly necessary ATM, but it's good practice ;)
- Reword all CSS variables to be more generic -> flexible (read my prev. comments).